src/Client/ClientBase.cpp

```diff
@@ -195,25 +200,21 @@ static void incrementProfileEventsBlock(Block & dst, const Block & src)
         }
     };
     std::map<Id, UInt64> rows_by_name;
+
     for (size_t src_row = 0; src_row < src.rows(); ++src_row)
     {
+        /// Filter out threads stats, use thead group stats
```

Comment:
Let's add a bit better comment here.
About why do we actually filer everything except thread_grop. (Like, metrics per thread are not needed, and now we do not send it, and it's only for compatibility now),